db/seeds.rb

```diff
@@ -0,0 +1,14 @@
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the rake db:seed (or created alongside the db with db:setup).
+#
+# Examples:
+#
+#   cities = City.create([{ name: 'Chicago' }, { name: 'Copenhagen' }])
+#   Mayor.create(name: 'Emanuel', city: cities.first)
+post1 = Post.create(body: "I like cake", user: "Tyler")
+post2 = Post.create(body: "Paste", user: "Mike")
+post3 = Post.create(body: "I like cake", user: "Trouble")
+
+Comment.create(body:"HAHA", post: post1)
+Comment.create(body:"HAHA", post: post2)
+Comment.create(body:"HAHA", post: post3)
```

Comment:
A much better way to do this is to use active record to associate the comments with the post
```rb
post1 = Post.create(body: "I like cake", user: "Tyler")

post1.comments.create(body: "HAHA")
```